lsst-epo/rubin-obs-client
# [F] Data products page

The data products page is a new entry type that combines two separate display types: the content blocks of pages with the media sidebar of news posts. This PR includes a refactor of existing sidebars into a single layout that is shared by News, Staff, Data Products pages to reduce style fragmentation.

API branch: `ag/page-sidebars`

To test:

- API
    - Create new custom breadcrumb in `Globals` called "For Scientists", linking the existing "For Scientists" page. The final setup will have an intermediary page called "Understanding Data and Simulations" with a slug of "data-products", but this can be left out for testing.
    - Create one or many "Data Products" pages. Add images, text, and other content blocks.

- Client
    - Verify display of sidebar on Staff and News post pages. Their display will slightly differ from what is on production since their existing styles are close but not exactly the same. End result intends to unify these styles with the current sidebar design shown in For Scientists page designs
    - Navigate to your data products page, verify that
       - has a sidebar with media inserted in content blocks
       - has a breadcrumb that shows the "For Scientists" link, and the intermediary page if added
       - has related "Data Products" 
       - has a link back to the intermediary page if added (otherwise will link back to "For Scientists")
